plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java

```diff
@@ -127,15 +129,40 @@ public PluginManager(Config cfg) {
         skipFailedPlugins = cfg.isSkipFailedPlugins();
         hashFunction = cfg.getHashFunction();
         httpClient = null;
+        try{
+            userAgentInformation = this.getUserAgentInformation();
+        }
+        catch(UserAgentInformationException e){
+            logVerbose("Unable to get the UserAgentInformation");
+        }
     }
 
+    private String getUserAgentInformation() throws UserAgentInformationException{
+        Properties properties = new Properties();
+        ClassLoader propertiesClassLoader = this.getClass().getClassLoader();
+        try(InputStream propertiesStream = propertiesClassLoader.getResourceAsStream("Version.properties")){
```

Comment:
file name should be lowercase
```suggestion
        try(InputStream propertiesStream = this.getClass().getResourceAsStream("version.properties")){
```

Reply:
> file name should be lowercase

Okay, I will change the file name into lowercase as well.